challenge-07/challenge-07.js

```diff
@@ -1,51 +1,68 @@
 /*
 Crie um array com 5 items (tipos variados).
 */
-// ?
+var arr = ['Felipe', 27, true, null, {cabelo: 'castanho'}];
 
 /*
 Crie uma função chamada `addItem`, que irá adicionar itens no array criado.
 A função deverá retornar o array atualizado.
 */
-// ?
+function addItem( param ){
+	arr.push( param );
+	
```

Comment:
-1
Vamos começar a aplicar _clean code_: só use espaços em branco quando for separar lógica :)